CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ Changes
 * [Developer]: Updated to Yarn v2.4.1.
 * [Developer]: Cleaned up compile warnings related to deprecated methods in Spring, JSON parsing, and Galaxy connectivity.
 * [UI] Updated And Design to v4.15.0.
+* [Database]: Removed `sample_metadata_entry` table which should have been dropped in 21.01 release. (21.01.3)
```

Comment:
Is the database change an old change?

Reply:
Yeah sorry I should have commented on that in the merge.  This branch was originally a hotfix and I pulled it back to being off `development`.  This changelog line was supposed to come over in the #961 hotfix PR.  It looks like it was missed in the development merge for that PR.  It needs to land in development somewhere either way.  It can either stay here or I can create a new PR just to merge that in.